packaging/update-source-packages.sh

```diff
@@ -21,9 +21,8 @@ if [[ ! -d "$SOURCE_DIR" ]]; then
     exit 1
 fi
 
-if [[ ! -d "$TARGET_DIR" ]]; then
```

Comment:
I think let's keep this check to accidentally stop you dumping things somewhere unexpected, I think we can do the `mkdir -p` though after it just to create the major version (it will do nothing if already present).

Reply:
make sense, changed it as recommended.